### PR TITLE
Update the governance.md document

### PR DESCRIPTION
**Pull request by kazydek**
**Description**

Changes proposed in this pull request:

- Added the current SIG/WG decision-making process description
- Updated the DR template to add information on the document naming convention 

**Related issue(s)**
Relates to #56

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -1,34 +1,61 @@
...
 
-## Maintainers' responsibilities
+## Ownership policy
 
-With great power comes great responsibility. Maintainers not only make changes and approve pull requests but also truly care about their projects.
+Kyma repositories are owned by code owners who are a group of people with special privileges in the repositories of the [GitHub](../../../) organization. Each repository has a separate `CODEOWNERS` file located at its root. The file specifies persons who have the ability to approve contribution to the part of the repository they own. after a detailed review of the related pull requests. Although the name suggests only the code ownership, the `CODEOWNERS` file is not only about the code but the content in general. Apart from the developers, you can define any relevant parties as code owners. For example, technical writers are set up as the owners of all `.md` documents in the Kyma repositories and SIG/WG members are the owners of their SIG/WG's folders' content.
```
Comment (klaudiagrz) [inline governance.md]: `The file specifies persons who have the ability to approve contribution to the part of the repository they own, after a detailed review of the related pull requests. ` -> changed dot in the middle of the sentence into a comma

File: governance.md [hunk 2]
```diff
@@ -1,34 +1,61 @@
...
+With great power comes great responsibility. Code owners not only review and approve pull requests but also truly care about their projects.
 
-## Become a maintainer
+Every code owner is expected to:
 
-To become a maintainer, contact one of the existing maintainers of the given project and agree to fulfill the requirements listed above.
-The existing maintainer can also propose a new member by creating a pull request adding the candidate to the `CODEOWNERS` file in the project's repository. The change requires the two-third majority of all maintainers' votes.
+* Contribute high-quality code and content
+* Communicate and collaborate with other code owners to improve the ownership process
+* Perform a thorough review of incoming pull requests and make sure it follows the contributing rules from the [`CONTRIBUTING.md`](CONTRIBUTING.md) document.
```
Comment (klaudiagrz) [inline governance.md]: `Perform a thorough review of incoming pull requests and make sure they follow the contributing rules`...

File: governance.md [hunk 3]
```diff
@@ -1,34 +1,61 @@
...
+### Add or remove a code owner
 
-The maintainer's role requires commitment and ongoing contribution to a project. If a maintainer is no longer able to fulfill these requirements or does not contribute to the project for half a year, remove them from the list. The change requires the two-third majority of all maintainers' votes.
+To suggest a change in the ownership of a given repository part, create a pull request with the required changes in the `CODEOWNERS` file in the project's repository. The required number of code owners needs to approve the pull request for the changes to take place. Read [here](https://github.com/kyma-project/community/blob/master/guidelines/internal-guidelines/repository-template/template/CODEOWNERS) how to set up and modify owners of the given repository folders and files.
+
+## The decisions-making process
+
+In general, the decisions that affect the project, including its structure, functionalities, components, or work of the project teams are made by the relevant Special Interest Groups (SIGs) and Working Groups (WGs). However, the organizational decisions and those that relate to the product strategy belong to the Kyma Council.
```
Comment (klaudiagrz) [inline governance.md]: `In general, the relevant Special Interest Groups (SIGs) and Working Groups (WGs) make the decisions that affect the project, including its structure, functionalities, components, or work of the project teams.`

File: governance.md [hunk 4]
```diff
@@ -1,34 +1,61 @@
...
+
+SIGs and WGs follow the "lazy consensus" decision-making process which assumes that:
+
+* All SIG/WG members have an equal voice in the decision-making process.
+* Silence is consent. By default, lack of objections to a proposed decision is interpreted as a silent approval.
```
Comment (klaudiagrz) [inline governance.md]: `By default, lack of objections to a proposed decision means a silent approval.`

File: governance.md [hunk 5]
```diff
@@ -1,34 +1,61 @@
...
+* Any objections are good opportunities for the discussion and healthy, constructive conflicts.
```
Comment (klaudiagrz) [inline governance.md]: `Any objections are good opportunities for the healthy, constructive discussion.` -> I wouldn't mention conflicts, I'm not sure whether they can be "healthy and constructive" and generally, this word invokes negative connotations.

File: governance.md [hunk 6]
```diff
@@ -1,34 +1,61 @@
...
+2. The PR creator sends the table with the decision log details to the related SIG/WG distribution list. If the decision input providers are not part of the mailing list, add them to the email communication.
+3. The discussion on the proposed decision can happen through the mailing list, on the related Slack channel, or directly on the pull request. You can also add the topic to the agenda of the upcoming SIG/WG meeting and discuss it with the SIG/WG members. Encourage the discussion and bring up any objections early in the process. A person with a different approach to the topic can create a Counter Proposal.
+4. The discussions on the proposed decision can lead to the change in the actual decision log or the proposal, or end up with no changes required. Update the pull request with the DR with the outcome of the discussion. Otherwise, take actions required to reach a consensus. Those who created a proposal work with those who had the objections to prepare an improved solution or reach a consensus to decline the proposal.
```
Comment (klaudiagrz) [inline governance.md]: 1. `lead to a change in the actual decision log or`~the~ `proposal` -> isn't there too many "the" ?
2. `Update the DR pull request with the outcome of the discussion.` -> too many "with" made the sentence clumsy
3. `Those who created the proposal`

File: governance.md [hunk 7]
```diff
@@ -1,34 +1,61 @@
...
+5. If you reach the "lazy consensus" by the decision-making date, update the status of the decision record with either `Accepted` or `Declined`, and merge the PR.
+6. If there are still unresolved objections by the decision due date, merge the PR with the `Proposed` status. The ultimate decision makers defined in the decision log table take the final decision.
+7. After the final decision, update the the decision log to either `Accepted` or `Declined`.
```
Comment (klaudiagrz) [inline governance.md]: `After the final decision, update` ~the~ `the decision log`

File: governance.md (hunk 3 above)
Comment (kazydek) [reply governance.md]: My original intention was to emphasize the type of decisions by mentioning them first, that is why ~I used passive voice here...but with the change in the active voice that also seems fine.

File: governance.md (hunk 5 above)
Comment (kazydek) [reply governance.md]: You're right. I've changed that.

File: governance.md (hunk 6 above)
Comment (kazydek) [reply governance.md]: 1. One per a noun so I guess that could stay...
2. I left `Update the PR with the outcome of the discussion.`
3. Ok

File: governance.md [hunk 8]
```diff
@@ -1,34 +1,61 @@
...
+
+In general, the relevant Special Interest Groups (SIGs) and Working Groups (WGs) make the decisions that affect the project, including its structure, functionalities, components, or work of the project teams. However, the organizational decisions and those that relate to the product strategy belong to the Kyma Council.
+
+SIGs and WGs follow the "lazy consensus" decision-making process which assumes that:
+
+* All SIG/WG members have an equal voice in the decision-making process.
+* Silence is consent. By default, lack of objections to a proposed decision means a silent approval.
+* Any objections are good opportunities for healthy and constructive discussions.
+
+> **NOTE:** The described approach only concerns the decisions taken during SIG and WG meetings. It does not affect any Kyma decisions taken during daily team activities.
```
Comment (Abd4llA) [inline governance.md]: since SIGs and WG do work all over the week not just during meetings, I would suggest to remove `meetings` from this sentence. Maybe reword around : only concerns the SIG and WG related decisions. OR: only concerns decisions being made as part of SIGs and WGs activities.

File: governance.md [hunk 9]
```diff
@@ -1,34 +1,61 @@
...
+
+### Process
+
+The SIG/WG decison-making process looks as follows:
+
+![Decision-making process](assets/decision-making-process.png)
+
+1. Every decision starts with creating a PR with the [**Decision Record**](./guidelines/templates/resources/DR.md) (DR) document that provides the decision details, timelines, status, the related context, and associated consequences. The DR can reference a proposal or any other related document.
+2. The PR creator sends the table with the decision log details to the related SIG/WG distribution list. If the decision input providers are not part of the mailing list, add them to the email communication.
```
Comment (Abd4llA) [inline governance.md]: suggest to change `distribution list` to `mailing list` .

File: governance.md [hunk 10]
```diff
@@ -1,34 +1,61 @@
...
+4. The discussions on the proposed decision can lead to a change in the actual decision log or the proposal, or end up with no changes required. Update the PR with the outcome of the discussion. Otherwise, take actions required to reach a consensus. Those who created the proposal work with those who had the objections to prepare an improved solution or reach a consensus to decline the proposal.
+5. If you reach the "lazy consensus" by the decision-making date, update the status of the decision record with either `Accepted` or `Declined`, and merge the PR.
+6. If there are still unresolved objections by the decision due date, merge the PR with the `Proposed` status. The ultimate decision makers defined in the decision log table take the final decision.
+7. After the final decision, update the decision log to either `Accepted` or `Declined`.
+
+>**NOTE:** There is a possibility to revisit the decision before the revision date specified in the decision log. To request it, provide details and the reason why you want to discuss the decision earlier, and raise an issue in the `community` repository.
```
Comment (Abd4llA) [inline governance.md]: We agreed that revisiting a decisions "before" the revisit date can only be issue by the ultimate decision maker.

File: governance.md (hunk 8 above)
Comment (kazydek) [reply governance.md]: `concerns the decisions made by SIGs and WGs.`
